packages/headless/src/controllers/facets/range-facet/date-facet/date-range.ts

```diff
@@ -47,8 +47,12 @@ export interface DateRangeOptions {
   useLocalTime?: boolean;
 }
 
-export function isSearchApiDate(date: string) {
-  return formatForSearchApi(dayjs(date)) === date;
+function isRelativeDate(rawDate: DateRangeInput): rawDate is RelativeDate {
+  return (
+    typeof rawDate !== 'number' &&
+    typeof rawDate !== 'string' &&
+    !(rawDate instanceof Date)
+  );
```

Comment:
Instead of determining a relative date by omission, it would be safer to check if it has a `period`, `units` etc.
Otherwise, if a new type is added to `DateRangeInput`, this function will need to be adjusted,